### PR TITLE
Fixes #25352: Event log rollback action is prevented from CSP headers

### PR DESCRIPTION
**Pull request by clarktsiory**
https://issues.rudder.io/issues/25352

There was some `onclick` event handlers that were dynamically assigned (`$(...).append('<button onclick=.../>')`), but the would never be executed since we have strict CSP headers.
We have to replace them by hand and bind within a `script` tag on the server template.

### CODE REVIEW

**Comment by clarktsiory** [comment]
Commit modified

File: webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/web/services/EventLogDetailsGenerator.scala [hunk 1]
```diff
@@ -109,7 +110,7 @@ class EventLogDetailsGenerator(
       val name = (x.details \ "rule" \ "displayName").text
       Text("Rule ") ++ {
         if (id.serialize.length < 1) Text(name)
-        else <a href={ruleLink(id)} onclick="noBubble(event);">{name}</a> ++ actionName
+        else <a href={ruleLink(id)}>{name}</a> ++ actionName
```
Comment (clarktsiory) [inline webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/web/services/EventLogDetailsGenerator.scala:113]: This `onclick` also causes issues, removing it still make it work fine

**Comment by clarktsiory** [comment]
PR updated with a new commit

**Comment by clarktsiory** [comment]
PR updated with a new commit

**Review by RaphaelGauthier — APPROVED** [review]

**Comment by clarktsiory** [comment]
OK, squash merging this PR

**Pull request merged**